apps/financial-aid/web-osk/src/components/Status/Approved.tsx

```diff
@@ -25,10 +31,16 @@ const Approved = ({ state, amount, isStateVisible }: Props) => {
       <Text as="h2" variant="h3" color="mint600" marginBottom={[4, 4, 5]}>
         Umsókn {getState[state].toLowerCase()}
       </Text>
-
-      <Text as="h3" variant="h3" marginBottom={2}>
-        Veitt aðstoð :{amount?.toLocaleString('de-DE') + ' kr.'}
-      </Text>
+      {!isUserSpouse ? (
```

Comment:
Wondering if we should revert this and have isApplicant? Like to have the default case rather then the edge case here.